components/public-api/buf.gen.yaml

```diff
@@ -14,7 +14,7 @@ plugins:
           - module=github.com/gitpod-io/gitpod/components/public-api/go
     - name: protoc-proxy-gen
       out: go
-      path: /workspace/go/bin/protoc-proxy-gen
+      path: /root/go-packages/bin/protoc-proxy-gen
```

Comment:
This is different between next and now config IIRC. We should align that in the image. :grimacing: